packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -364,13 +366,27 @@ export function getResolver({
              */
             if (response.headers['content-type'].includes('application/json')) {
               body = JSON.parse(body)
+            } else {
+              /**
+               * If not application/json, assume no additional processing is
+               * needed
+               *
+               * TODO: Should this also make a check to see what is defined in
+               * the schema? Probably not, would be an OAS/API mismatch.
+               */
+              resolve(body)
```

Comment:
This is already covered by a test case FYI. We were supposed to resolve here but we didn't. Luckily, it didn't break anything before.